src/katsdpcontroller/product_controller.py

```diff
@@ -1497,7 +1497,15 @@ async def beam_delays(self, stream_name: str, coefficient_sets: Sequence[str]) -
             timeout=DELAYS_TIMEOUT,
         )
 
-    async def dsim_signals(self, dsim: str, signals_str: str, period: Optional[int] = None) -> int:
+    def _get_dsim_katcp(self, dsim: str) -> aiokatcp.Client:
+        """Get the katcp client for connecting to a dsim.
+
+        Raises
+        ------
+        FailReply
+            if the dsim name is not known, there is no connection to it, or the
+            subarray product is not in a suitable state.
+        """
```

Comment:
What's the protocol for including a parameter description for a param that will _always_ come from a public-facing, documented call?
* Basically, I previously conceded we didn't need to document `dsim: str` param in `dsim_signals` because it came from `request_dsim_signals`

Would you be opposed to adding that same description here?
```
Parameters
------------
dsim
    The task name for the dsim (implementation-defined).
```

Reply:
I'm going to vote for the docstring update to include the parameter description. Just for anyone else/implementing the next `dsim-request`, you don't need to consult existing call chains on what exactly it's expecting.